quickbooks_endpoint.rb

```diff
@@ -226,6 +226,19 @@ class QuickbooksEndpoint < EndpointBase::Sinatra::Base
     result code, summary
   end
 
+  post '/get_payments' do
+    qbo_payment = QBIntegration::Payment.new(@payload, @config)
+    summary, page, since, code = qbo_payment.get()
+
```

Comment:
Indentation woes here as well. Not a big problem, just makes confusing to check whether methods are correctly closed, might lead to bugs.

Reply:
@gorandev eagle eyes! I'm glad you're as sadistic as I am for this kind of thing. I can't believe I didn't notice until now